android/app/build.gradle

```diff
@@ -49,7 +49,7 @@ android {
         applicationId "com.example.star_movie_app"
         // You can update the following values to match your application needs.
         // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-build-configuration.
-        minSdkVersion flutter.minSdkVersion
+        minSdkVersion localProperties.getProperty('flutter.minSdkVersion').toInteger()
```

Comment:
why did u change it?

Reply:
Because min 19 sdk version is required, didn't want to hardcore